### DISCUSSION TITLE
Compton Scattering?

### DISCUSSION THREAD

**Discussion by TheHamiltonian** [epochpic/epoch#171] [Ideas]
Is the compton scattering feature planned? I'm not sure whether QED block actually contains such dynamics. This could  enable simulations similar to the one recently published with OSIRIS

https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.125.265001

**Comment by TomGoffrey** [comment]
I don't believe Compton scattering is included in the current code, although my understanding from Ridgers et al 2014 is that it would be possible to include the effects in the QED module. That being said, I'm not familiar with the Osiris work.

Unfortunately this development isn't on our current list of funded development of the EPOCH code, and it also doesn't fall within any of the existing projects we're currently working on, so it won't be added.

Such a development would have to happen as part of a new research project. If you're interested in collaborating then the best thing to do is to email myself or one of the other developers. Alternatively, EPOCH is open source, and we'd certainly be happy to accept pull requests.

**Comment by TomGoffrey** [comment]
I realise this is very old, but I am currently in the process of adding linear Compton scattering to EPOCH, and it will be included inthe next release. If you are interested in testing this, let me know.

**Reply by mlamac** [reply]
Yes, I am still interested in testing this. Feel free to send me more information on my email marcel.lamac@eli-beams.eu

**Reply by TomGoffrey** [reply]
Thanks. Sorry for the slow reply, I'm just trying to finalise the implementation before handing it over. I will aim to email you in the next few days.
